st2actions/st2actions/cmd/st2notifier.py

```diff
@@ -49,6 +49,7 @@ def _run_worker():
     actions_notifier = notifier.get_notifier()
     try:
         actions_notifier.start()
```

Comment:
suggestion : can add a single start_and_wait() function in MessageHandler. Simplifies this code a little bit.